packages/pwa/app/hooks/use-auth-modal.js

```diff
@@ -222,7 +225,7 @@ export const AuthModal = ({
                         <ResetPasswordForm
                             form={form}
                             submitForm={submitForm}
-                            clickSignIn={() => setCurrentView(LOGIN_VIEW)}
+                            clickSignIn={onBackToSignInClick}
```

Comment:
On line 221, should the value be `onBackToSignInClick` too?

Reply:
Yea you are right 👍